imports/blocks/give/fieldsets/shared/monetize.js

```diff
@@ -0,0 +1,24 @@
+const monetize = (amount, fixed) => {
```

Comment:
totally. that's why i just left it here and didn't abstract it all the way out yet.